app/models/collection.rb

```diff
@@ -5,4 +5,10 @@
 class Collection < ApplicationRecord
   has_many :works, dependent: :destroy
   belongs_to :creator, class_name: 'User'
+
+  validates :contact_email, presence: true
```

Comment:
Did/would you consider making these one-liners?

```ruby
validates :contact_email, :description, :managers, :name, :visibility, presence: true
```

:question: More readable? Less readable? I could go either way, tbh. What do you think?